src/pymor/operators/fv.py

```diff
@@ -467,3 +468,205 @@ def _assemble(self, mu=None):
         F_INTS /= g.volumes(0)
 
         return F_INTS.reshape((1, -1))
+
+
+class DiffusionOperator(NumpyMatrixBasedOperator):
+    """Linear finite volume diffusion |Operator| using finite differences.
+
+    The operator is of the form ::
+
+    L(u, mu)(x) = ∇ ⋅ (mu ∇ u(x))
+
+    Parameters
+    ----------
+    grid
+        |Grid| over which to assemble the operator.
+    boundary_info
+        |BoundaryInfo| determining the Dirichlet and Neumann boundaries.
+    diffusion
+        Diffusion scalar 'mu'.
+    name
+        The name of the operator.
+    """
+
+    sparse = True
+
+    def __init__(self, grid, boundary_info, diffusion=1.0, name=None):
+        super(DiffusionOperator, self).__init__()
+        self.grid = grid
+        self.boundary_info = boundary_info
+        self.diffusion = diffusion
+        self.name = name
+        self.source = self.range = NumpyVectorSpace(grid.size(0))
+
+    def _assemble(self, mu=None):
+        diffusion = self.diffusion
+        grid = self.grid
+        assert isinstance(grid, AffineGridWithOrthogonalCentersInterface)
+        centers = grid.centers(1)
+        orthogonal_centers = grid.orthogonal_centers()
+        boundary_mask = grid.boundary_mask(1)
+        VOLS = grid.volumes(1)
+        DVOLS = np.zeros(VOLS.size)
+
+        SE_I0 = grid.superentities(1, 0)[:, 0]
+        SE_I1 = grid.superentities(1, 0)[:, 1]
+
+        SE_I0_I = SE_I0[~boundary_mask]
+        SE_I1_I = SE_I1[~boundary_mask]
+
+        DVOLS[~boundary_mask] = np.abs(orthogonal_centers[SE_I0_I, 0] - orthogonal_centers[SE_I1_I, 0])
+        DVOLS[~boundary_mask] += np.abs(orthogonal_centers[SE_I0_I, 1] - orthogonal_centers[SE_I1_I, 1])
```

Comment:
If I am not mistaken, you are calculating the 1-norm of the difference vector between the two orthogonal_centers on each side of the edge. But shouldn't it be the euclidean norm? That would be
```
DVOLS[~boundary_mask] = np.linalg.norm(orthogonal_centers[SE_I0_I, :] - orthogonal_centers[SE_I1_I, :], axis=1)
```